const fail = (error: unknown) => Promise.reject(error)

export { fail }
